src/CommonRLInterface.jl

```diff
@@ -58,4 +75,7 @@ reset!(env::RLBaseEnv) = CRL.reset!(env.env)
 (env::RLBaseEnv)(a) = env.r = CRL.act!(env.env, a)
 Base.copy(env::CommonRLEnv) = RLBaseEnv(CRL.clone(env.env), env.r)
 
-ActionStyle(env::RLBaseEnv) = CRL.provided(CRL.valid_actions, env.env) ? FullActionSet() : MinimalActionSet()
\ No newline at end of file
+ActionStyle(env::RLBaseEnv) = CRL.provided(CRL.valid_actions, env.env) ? FullActionSet() : MinimalActionSet()
+UtilityStyle(env::RLBaseEnv{<:CRL.AbstractZeroSumEnv}) = ZERO_SUM
+InformationStyle(env::RLBaseEnv) = IMPERFECT_INFORMATION
+InformationStyle(env::RLBaseEnv{<:CRL.AbstractMarkovEnv}) = PERFECT_INFORMATION
```

Comment:
As of now, CommonRLInterface doesn't have the traits necessary to query whether `AbstractMarkovEnv` is an MDP or POMDP - it seems like this should be `IMPERFECT_INFORMATION`.

Reply:
I see. I agree that `IMPERFECT_INFORMATION` is a better default value.